tests/ui/async-await/async-closures/without-precise-captures-we-are-powerless.rs

```diff
@@ -38,10 +38,12 @@ fn through_field_and_ref<'a>(x: &S<'a>) {
     let c = async || { println!("{}", *x.0); }; //~ ERROR `x` does not live long enough
     outlives::<'a>(c());
     outlives::<'a>(call_once(c)); //~ ERROR explicit lifetime required in the type of `x`
+}
```

Comment:
Split into two functions since one error actually just suppresses the other (due to NLL error deduplication, I believe).